contracts/dao-dao-core/src/contract.rs

```diff
@@ -106,10 +106,15 @@ pub fn execute(
     info: MessageInfo,
     msg: ExecuteMsg,
 ) -> Result<Response, ContractError> {
-    // No actions can be performed while the DAO is paused.
+    // Check if the DAO is paused
     if let Some(expiration) = PAUSED.may_load(deps.storage)? {
         if !expiration.is_expired(&env.block) {
-            return Err(ContractError::Paused {});
+            // If paused, then only allow messages from the Admin or DAO itself
+            if info.sender != env.contract.address
+                && Some(info.sender.clone()) != ADMIN.may_load(deps.storage)?
```

Comment:
```suggestion
                && info.sender.clone() != ADMIN.load(deps.storage)?
```

`ADMIN` should always be defined, so it's safe to just use `ADMIN.load` here i'm pretty sure